client/settings/payment-method-settings/index.js

```diff
@@ -71,13 +76,19 @@ const PaymentMethodSettings = ( { methodId } ) => {
 			<Banner />
 
 			<h2 className="payment-method-settings__breadcrumbs">
-				<a href={ getPaymentSettingsUrl() }>WooCommerce Payments</a>{ ' ' }
+				<a href={ getPaymentSettingsUrl() }>
+					{ __( 'WooCommerce Payments', 'woocommerce-payments' ) }
+				</a>{ ' ' }
 				&gt; <span>{ title }</span>
 			</h2>
 
 			<SettingsSection Description={ Description }>
-				<Controls />
+				<LoadableBlock isLoading={ isLoading } numLines={ 30 }>
+					<Controls />
+				</LoadableBlock>
 			</SettingsSection>
+
+			<SaveSettingsSection />
```

Comment:
Save button moved from the `client/settings/payment-method-settings/digital-wallets-settings.js` to here